docs/blog/newsletter-february-2021.rst

```diff
@@ -0,0 +1,88 @@
+
+.. post:: February 03, 2021
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – February 2021
+#########################################
+
+Hey everyone! It's a month into 2021 already and the newsletter team is back in business. We've got a packed issue for you today, with topics ranging from screenshots to embedded code snippets to showing your value, so I'll get right on with it.
+
+Just a couple announcements today: the East Coast Quorum meetup is holding its first virtual event this month! It's on "Lessons learned open-sourcing GitHub Docs", and it'll be on Monday 22nd February at 19:30 Eastern Time. If you're interested, `sign up to attend <https://www.meetup.com/virtual-write-the-docs-east-coast-quorum/events/276054186/>`__. We also have a proposed Write the Docs Enhancement Proposal (WEP) focused on Slack moderation. There is an `open pull request <https://github.com/writethedocs/weps/pull/4>`__ with the proposal for ``WEP2``, and the community feedback period lasts until March 5.  
```

Comment:
Just a note -- I added the WEP here since it's ready to go out. /cc @bethaitman

Reply:
Cool, thanks for letting me know.